src/components/select-field/__tests__/MultiSelectField.test.js

```diff
@@ -10,7 +10,7 @@ describe('components/select-field/MultiSelectField', () => {
     ];
 
     const intl = {
-        formatMessage: jest.fn(),
+        formatMessage: jest.fn().mockReturnValue('Clear All'),
```

Comment:
This should probably be mocked closer to where we expect this value instead of for everything. also consider using [`mockImplementationOnce`](https://jestjs.io/docs/en/mock-function-api.html#mockfnmockimplementationoncefn)